packages/editor/src/client/locales/en.json

```diff
@@ -724,6 +724,9 @@
       "nextSaturday": "Next Saturday",
       "hour": "{{hour}}:00"
     },
+    "guideTour": {
```

Comment:
We should have all the text in the translations. I would start a new object on the top level and use that for all the guided tour translations.